Documentation/Testing/TestRunners.rst

```diff
@@ -0,0 +1,387 @@
+..  include:: /Includes.rst.txt
+..  index:: Testing; Project; Runners; runTests; make; TestRunners
+..  _testing-projects-organization:
+
+========================================
+Test Runners: Organize and execute tests
+========================================
+
+Executing commands to run tests and other toolchain scripts
+can be tedious. Sadly there is no "one-fits-all" solution for this.
+
+This chapter tries to explain available options, in summary:
+
+*   Use a bash alias
+*   Write a bash script
+*   Introduce `custom DDEV commands <https://ddev.readthedocs.io/en/stable/users/extend/custom-commands/#custom-commands>`__
+*   Introduce a `Composer script <https://getcomposer.org/doc/articles/scripts.md>`__
+*   Create a Makefile
+*   Use dedicated tools like `just <https://github.com/casey/just>`__
+*   Use the :file:`runTests.sh` script based on the TYPO3-Core
+*   Use a customized :file:`runTests.sh` script based on `blog_example`
+*   Use a generator
+
+All these commands and tasks are not only run by you as a
+developer (or end-user) and your co-workers, but are also
+relevant for Continuous Integration (CI),
+Continuous Deployment (CD) - for example via GitHub Actions
+or GitLab Pipelines.
+
+..  _testing-projects-organization-why:
+
+What are test runners and why do we need them
+---------------------------------------------
+
+As you can read in the chapters about :ref:`Testing <testing>`, there is a multitude
+of available tooling around a project:
+
+*   Unit testing
```

Comment:
Should we link those that already have chapters?

Reply:
I would link them. On the Internet you cannot force ppl to keep reading. And if they have to search for a definition of unit tests at this point it is more distracting then a link. Also ppl might have landed on content that is still to complicated for them